.github/CODEOWNERS

```diff
@@ -445,6 +445,7 @@
 /pkg/flare/*_win.go                     @Datadog/windows-agent
 /pkg/flare/*_windows.go                 @Datadog/windows-agent
 /pkg/flare/*_windows_test.go            @Datadog/windows-agent
+/pkg/flare/sysprobe                     @DataDog/Networks
```

Comment:
suggestion - as sysprobe is very generic, let's add all system-probe teams as owners to the directory
or let's rename this package to indicate it belongs only with NPM

Reply:
Thanks, added other system-probe teams as owners - did I miss anybody?